README.md

```diff
@@ -91,6 +91,27 @@ make -s local print-env > .env
 
 To run the command above you need [AWS credentials](#aws-credentials).
 
+#### Git secrets
```

Comment:
I think that the main README file should contain the minimal information to let you start develop. This information can be in a separate Markdown document in the `/documents` folder

Reply:
Fair point. I would want to ensure that all new joiners do set this up though. So shall I add it to the documentation folder and then link it from the README?

Reply:
Yes I think it is better in that folder with just a link from the README